changelogs/unreleased/gh-9825-space-format-default-func-tuple-arg.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* Added the source tuple as a second argument to the `default_func` field
+  option of `space:format` (gh-9825).
```

Comment:
Sounds confusing (Added the source tuple to the field option of space:format), I'd rewrite it a bit more detailed.

Reply:
Done.